lib/hashie/extensions/dash/indifferent_access.rb

```diff
@@ -11,6 +11,7 @@ module ClassMethods
           # Check to see if the specified property has already been
           # defined.
           def property?(name)
+            name = defined?(translations) && translation_exists?(name) ? translations[:"#{name}"] : name
```

Comment:
Is L25 written like that for a reason? I can't think of one, you should change that too.